post-policy.go

```diff
@@ -152,6 +152,23 @@ func (p *PostPolicy) SetCondition(matchType, condition, value string) error {
 	return errInvalidArgument("Invalid condition in policy")
 }
 
+// SetTagging - Sets tagging for the object for this policy based upload.
+func (p *PostPolicy) SetTagging(tagging string) error {
+	if strings.TrimSpace(tagging) == "" || tagging == "" {
```

Comment:
`|| tagging == ""` seems to be redundant

Reply:
I see that it's done more often in this file. I'm not a big fan of checking these trimmed values. It's not very likely that clients will issue a space or a newline, so the check is almost never necessary. And if they do specify it, then the check will fail on the back-end. I would prefer to remove all `string.TrimSpace(..)` in this file.

Reply:
I have no idea. @harshavardhana @klauspost  any advice?